lib/url-assembler-factory.js

```diff
@@ -83,6 +83,8 @@ module.exports = function (request) {
     if (typeof key === 'object') {
       return _multiParam(this, key, (value === true));
     }
+    key = encodeURI(key);
+    value = encodeURI(value);
```

Comment:
`/` param still not encoded. 
[encodeURI()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI "encodeURI()") does't encode `/`.  [encodeURIComponent()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent "encodeURIComponent()") does.

@mickaeltr Is this expected behavior?

```js
const url = UrlAssembler('http://example.com')
    .template('search/:q')
    .param({
        q: "/"
    }).toString();
console.log(url); // http://example.com//
```